src/cmd.ts

```diff
@@ -297,7 +297,25 @@ program
 		});
 		await db.destroy();
 	});
-
+program
+	.command("clear-indexer-failures")
+	.description("Clear the cached details of indexers (failures and caps)")
+	.action(async () => {
+		console.log(
+			"\nIf you've received a '429' (rate-limiting), continuing to hammer",
+			"your indexers may result in negative consequences.",
```

Comment:
```suggestion
			"If you've received a '429' (rate-limiting), continuing to hammer",
			"your indexers may result in negative consequences.",
```

Reply:
It appears like on the very next line and i wanted it to stand out more so I added the \n

not a big deal though.